main.go

```diff
@@ -54,6 +54,7 @@ var (
 
 	samplesQueueCapacity = flag.Int("storage.incoming-samples-queue-capacity", 64*1024, "The capacity of the queue of samples to be stored. Note that each slot in the queue takes a whole slice of samples whose size depends on details of the scrape process.")
 
+	chunkType       = flag.Int("storage.local.chunk-type", 1, "Which chunk encoding version to use. Currently supported is 0 and 1.")
```

Comment:
Is it worth exposing this as a flag? Are there realistic scenarios where we want to prefer single-delta over double-delta? If not, I would just hardcode this decision in the storage to always try to pick the best chunk type (either statically or dynamically based upon the specific data at some point).